lib/cloudshaper/cli.rb

```diff
@@ -0,0 +1,98 @@
+require 'thor'
+require 'cloudshaper'
+
+module Cloudshaper
+  class CLI < Thor
+    class_option "template_dir", type: "string", default: "templates"
+    class_option "remote_state", type: "boolean"
```

Comment:
We should probably add an option `--terraform-binary` so people can specify where it lives if it's not in $PATH